src/main/java/tqs/example/impostor/repository/Order.java

```diff
@@ -15,6 +15,10 @@ public class Order {
     @JoinColumn(name = "acp_id", nullable = false)
     private ACP acp;
 
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "locker_id", nullable = false)
```

Comment:
Locker and ACP data attributes need to be nullable. This has to be changed both for locker and acp in Order class.